frontend/frontend-filters-links.php

```diff
@@ -469,6 +469,11 @@ protected function get_queried_term_id( $tax_query ) {
 			return $term;
 		}
 
+		// Avoid a useless DB request if the taxonomy is not translatable.
+		if ( ! $this->model->is_translated_taxonomy( $taxonomy ) ) {
+			return $term;
+		}
```

Comment:
You are changing the conditional logic from the previous version:

```php
$obj = $wp_query->get_queried_object();
if ( ! empty( $obj ) && $this->model->is_translated_taxonomy( $taxonomy ) ) {
    $language = $this->model->term->get_language( $obj->term_id );
}
```

The previous version would NOT call to `PLL_Translated_Term::get_language()` if the taxonomy is not translatable.

Here it would call to `PLL_Translated_Term::get_language()` with a slug parameter instead.

Reply:
Yes I know. But it has no impact. The final result should be the same.